common/utils/start.sh

```diff
@@ -41,7 +41,22 @@ then
     fi
 fi
 
-# Set up the env for the spark user
+# Determine SPARK_CONF_DIR. If a non-empty config has been given then use it
+ls -1 /etc/oshinko-spark-configs &> /dev/null
```

Comment:
what if we just make it an either/or with the default behavior being to use `/etc/oshinko-spark-configs` ?